{
  "name": "CVE-2019-15486",
  "seq": "2019-15486",
  "type": "CAN",
  "status": "Candidate",
  "phase": {
    "text": "Assigned",
    "date": "20190822"
  },
  "desc": "django-js-reverse (aka Django JS Reverse) before 0.9.1 has XSS via js_reverse_inline.",
  "refs": [
    {
      "text": "https://github.com/ierror/django-js-reverse/compare/v0.9.0...v0.9.1",
      "source": "MISC",
      "url": "https://github.com/ierror/django-js-reverse/compare/v0.9.0...v0.9.1"
    },
    {
      "text": "https://github.com/ierror/django-js-reverse/pull/81",
      "source": "MISC",
      "url": "https://github.com/ierror/django-js-reverse/pull/81"
    }
  ],
  "votes": {}
}
